R/process_data.R

```diff
@@ -197,8 +197,8 @@ process_price_data <- function(data, technologies, sectors, start_year, end_year
       expected_levels_list =
         list(
           year = start_year:end_year,
-          ald_sector = sectors,
-          technology = technologies,
+          ald_sector = .env$sectors,
```

Comment:
I'm always a bit surprised by when to use `.env$` .. I usually get errors when using it outside a pipe.. but I'm guessing you did not get any messages